src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -83,7 +124,12 @@ def run(self, *args, **kwargs):
         pass
 
 class pt5(Macro):
-    """Macro with a motor parameter followed by a list of numbers"""
+    """Macro with a motor parameter followed by a list of numbers.
+    Usages, ex.:
+    pt5 mot1 [[1][3]]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.